src/gui/editors/SongEditor.cpp

```diff
@@ -808,7 +829,7 @@ void SongEditor::updatePosition( const MidiTime & t )
 	if( x >= trackOpWidth + widgetWidth -1 )
 	{
 		m_positionLine->show();
-		m_positionLine->move( x, m_timeLine->height() );
+		m_positionLine->move( x-7, m_timeLine->height() );
```

Comment:
A bit of a nitpick, but could the 7 (and the 8 above) be either 1) sourced from the theme or 2) made to use the same constant variable above? This is the kind of magic number that can cause bugs, i.e. where two separate magic numbers need to have a certain relationship to be valid.

Reply:
> the number here must always be one less than the number above.

Yes, that's right. Fixed it in the new commit, along with other things.